### ISSUE TITLE
Ilen contribution

### ISSUE THREAD

**Issue by perich** [ThorGroup/FinalProject#5]
Hi @ilentt 

A few of us in the group are very concerned about your contribution to the group. We have not seen any code or contribution yet and we have just 1 week left in the project.

You dont respond to our facebook chat.

We would like for you to commit all of your code on a branch here, so that we can review it, and @lingboo can try to complete it in time.

Thanks,
Peter

**Comment by ilentt** [comment]
Hi @perich 

My code have merge with master branch before and I am working hard to complete my current task on time. I planed submit my contribution on this Sunday with support from @lingboo and @tanlnm512 

Thanks,

I Len

**Comment by lingboo** [comment]
ok @ilentt, can you go to the final tutorial tomorrow ? and we will do it
together

**Comment by perich** [comment]
Ok. @lingboo has offered to help a lot, but he is concerned because you havent contacted him or told him what to do to help you.

We are both concerned that you are more worried about how this project can help you after the class, and how you can get it on your sever.

But right now, our only focus should be to get this project completed and working for presentation day. Right now, all the parts are very close to completed and working, except yours.

You say that your part will be finished by sunday night? So then we can take a picture (or upload it). We can add a status, we can choose a category, we can choose a reaction true/false, and we can submit it to the server together (I will take care of the HTTP part of submission). 

This can all be finished and we can test it tomorrow night?

**Comment by ilentt** [comment]
@lingboo We are going on tutorial tomorrow to work together to complete task.

@perich As I said before I am try my best to compete my task. Please give me some time to complete, and I appreciate your understanding.

Thanks.

**Comment by perich** [comment]
@ilentt I understand you need time to complete, you are busy with work. But we are all just as busy with work as you, and everyone else's part of this project has been the same or larger than yours, so you dont get my sympathies. We all have the same time to complete this.

This was supposed to be finished or nearly finished by last tuesday. Yet last tuesday you still didnt quite understand what your task was. Why did you ask me or someone else before that? Why do you almost never contribute on our FB chat?

I am personally very concerned with your attitude. On Thursday, you are more concerned how you can get the backend on your sever, than you are about completing you part of the project. You keep asking how this can help you, how it can help with your job. But right now, we just need to focus on finishing it.

I hope you can prove me wrong. And I hope you and @lingboo can complete this tomorrow. But when you ask for my sympathies and appreciate about time, you will not get it. I dont like excuses. 

@lingboo I want to put you in charge of this portion of the project. I know you are a very reliable member of our team, you make no excuses, and you want to finish this project and make it great. I hope @ilentt will explain what he has done, and what still needs to be finished. Can I count on you to do this @lingboo ? Let me know if you need my help.

**Comment by ilentt** [comment]
@perich I know how you feel caused by me. However this time is not the time to argue, we should focus on completing the project in the best possible, other issues you can talk directly to me if you want. So, please do not judge, let me try to finish my task on the best.

Thanks.

**Comment by lingboo** [comment]
@perich let him time to do, i'm sure that we will finish in tomorrow, and
right now we need to keep silent and do our best

**Comment by perich** [comment]
@lingboo You asked _me_ to talk to ilen because _you_ were concerned with his lack of contribution. Now you say I should be silent and let him work. I think you know more about what is going on that I do, so I will leave you to lead this group.

I will have my part finished by Monday afternoon. I will submit the code to you for your approval and let you add it to the project. I will keep silent from now on.

**Comment by lingboo** [comment]
You understand wrong way that I want to say Peter, you asked I len and he responded, we have tonight, tomorrow, and i'm sure that will be finished , trust me. You are getting angry and you need to calm down. The best way to do right now is keep calm, and do it. Please feel it for me

**Comment by ilentt** [comment]
I think what his meant is that you have the right to criticize me if I did something wrong. However, we should focus on completing our work well also other issues we will discuss later.

It's just that there are some misunderstandings between team members. Maybe we should go for a drink together someday to resolve this misunderstanding as the men. 

Ha ha... :joy:

**Comment by perich** [comment]
@lingboo I'm not angry, just frustrated. You say one thing on Facebook, and different thing here. @ilentt I'm criticizing anyone. I just think we need to be more professional. 

I just want to get this project finished, and we present it. I hope is something we can be proud of in the end.

Once that is all over, and this project is behind us, I'd be happy to get that drink with you all. I think you're all great guys, I just think we need to take this seriously for one more week. Then we can forget about it, and go back to other work :)

**Comment by lingboo** [comment]
@perich please see again conversation. Now focus on the final project, and forget all misunderstanding

**Comment by perich** [comment]
@lingboo Yes, I read convo. [http://imgur.com/a/W5LkS](here is the convo, in case you forget it too.) You were concerned about ilen, and wanted me to contact him via Github, so I dont know why said I should be silent.

Yes, we will focus on final project. I have no fear about anything now, I am confident in you as final project leader. You will do a great job. Let me know if you need help. :)

**Comment by lingboo** [comment]
@perich i mean just keep silent when can't keep calm. About conversation,
when I len reply and explain what he will do with final project i feel
better and sympathy. That's all. We can stop now and keep on work
